src/main/java/net/syscon/elite/repository/jpa/repository/OffenderNonAssociationDetailRepository.java

```diff
@@ -0,0 +1,10 @@
+package net.syscon.elite.repository.jpa.repository;
+
+import net.syscon.elite.repository.jpa.model.OffenderNonAssociationDetail;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface OffenderNonAssociationDetailRepository extends CrudRepository<OffenderNonAssociationDetail, OffenderNonAssociationDetail.Pk> {
```

Comment:
Not sure if both these repos are strictly needed.  Easy to remove if not though.

Reply:
Well seems like you can retrieve the OffenderNonAssociation from the OffenderNonAssociationDetail so having one repository might better communicate which entity is the most important.

Can you have a OffenderNonAssociation without a OffenderNonAssociationDetail?

Reply:
I originally had the join the other way from non-associations to non-association details but it didn't really marry up with how the legacy DB was structure.  This is the one likely to stay in all honesty...